synapse/config/emailconfig.py

```diff
@@ -250,18 +252,24 @@ def default_config(self, config_dir_path, server_name, **kwargs):
         #   #
         #   # If this option is set to false and SMTP options have not been
         #   # configured, resetting user passwords via email will be disabled
+        #   #
         #   #trust_identity_server_for_password_resets: false
         #
         #   # Configure the time that a validation email or text message code
         #   # will expire after sending
         #   #
         #   # This is currently used for password resets
+        #   #
         #   #validation_token_lifetime: 1h
         #
         #   # Template directory. All template files should be stored within this
         #   # directory
         #   #
-        #   #template_dir: res/templates
+        #   # If not set, the default location is `(location of Synapse's
+        #   # virtualenv)/res/templates`. Using an absolute path is recommended
```

Comment:
why do we specifically recommend an absolute path for this setting, and not for any of the other path settings in the config?